City-of-Helsinki/helfi-etusivu-news-search
# Uhf 5917 pagination

# [UHF-5917 Pagination component](https://helsinkisolutionoffice.atlassian.net/browse/UHF-5917)

## What was done?
- Add new component for pagination
- Add classes matching HDBT component
- Add function for scrolling to top of results when changing page

## How to install
- Make sure your elastic-container is up and running in local Etusivu environment and you have data indexed
- `nvm use; npm i; npm start`

## How to test
- Try clicking around the component and see if it works intuitively
- To see what the component looks like with more pages, change the `size` prop to 1 (Line 65 in `src/containers/ResultsContainer.tsx`
